### ISSUE TITLE
AUX loss WEIGHT and RATIO

### ISSUE THREAD

**Issue by bhack** [yoxu515/aot-benchmark#72]
Can you clarify a bit the role in the default config of:

```python
self.TRAIN_AUX_LOSS_WEIGHT = 1.0
self.TRAIN_AUX_LOSS_RATIO = 1.0
```

**Comment by z-x-yang** [comment]
aux_loss is the segmentation loss on the reference frame. Weight is the loss weight. Ratio is when to stop the aux_loss during training. Ratio=1 denotes it will not be stopped.在 2023年12月5日，20:35，bhack ***@***.***> 写道：﻿
Can you clarify a bit the role in the default config of:
self.TRAIN_AUX_LOSS_WEIGHT = 1.0
self.TRAIN_AUX_LOSS_RATIO = 1.0

**Comment by bhack** [comment]
>  Ratio=1 denotes it will not be stopped.

But I see `aux_weight = self.aux_weight * max(self.aux_step - step,
                                           0.) / self.aux_step`
                                           
And `self.aux_step = cfg.TRAIN_TOTAL_STEPS * cfg.TRAIN_AUX_LOSS_RATIO + 1e-5`

so also at Ratio 1 it is going to change the weight over time right?

What policy it is suggested for finetuning?

**Comment by bhack** [comment]
I am asking this cause finetuning your full checkpoints I always see a bump in the loss/MIOU at `cfg.TRAIN_TOTAL_STEPS/2`
